examples/lfric/eg17/README.md

```diff
@@ -0,0 +1,54 @@
+# PSyData Examples
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+This directory contains some runnable examples of LFRic. They are based
+on a subset of the LFRic infrastructure library, that has been slightly
+modified to make it easier to create stand-alone, non-MPI LFRic codes.
+The examples here show:
+- A [full_example](./full_example) of a stand-alone LFRic-based code. It
+  uses PSyclone for a very simple kernel.
+- [Extraction](./full_example_extract) of input- and output-parameter
+  of a kernel to NetCDF files.
+- A stand-alone example using [NetCDF](./full_example_netcdf) to read
```

Comment:
Again, see the similar comments above about providing more information (also valid for the mesh name in this case).

**Edit:** I see that "Detailed instructions are in the various subdirectories." I assume you mean the relevant subdirectories in `eg17`? I checked the relevant `README`  and it could use some more information. I suggest adding the information requested for this bullet point (if it is not already there) to the `full_example_netcdf/README.md`.

Reply:
As above :)